test/unit/lib/plugins/aws/package/compile/events/stream.test.js

```diff
@@ -375,6 +375,7 @@ describe('AwsCompileStreamEvents', () => {
                   arn: 'arn:aws:dynamodb:region:account:table/bar/stream/2',
                   batchWindow: 15,
                   maximumRetryAttempts: 4,
+                  functionResponseType: ['ReportBatchItemFailures'],
```

Comment:
As mentioned in my previous message, let's not extend these old tests, but instead extend or introduce a new one based on `runServerless` utility.

Reply:
Sorry about that. Need to look at the docs and see how it works.

Reply:
No worries - let me know if you have any questions around that :+1:

Reply:
@pgrzesik Do we have a new folder to place the new tests? or is it under the same folder?

Reply:
We already have a few of those in a separate describe in this file - please see: https://github.com/serverless/serverless/blob/b81fd0557f53e11597fe67bb138c09675d61e0c5/test/unit/lib/plugins/aws/package/compile/events/stream.test.js#L1585

Reply:
Thanks :)